scalafix-cli/src/main/scala/scalafix/internal/v1/Args.scala

```diff
@@ -376,11 +376,15 @@ case class Args(
 
   def configuredGlobal: Configured[LazyValue[Option[Global]]] =
     Configured.ok {
-      val settings = new Settings()
-      settings.YpresentationAnyThread.value = true
-      settings.classpath.value = validatedClasspath.syntax
-      val reporter = new ConsoleReporter(settings)
-      LazyValue.fromUnsafe(() => new Global(settings, reporter))
+      LazyValue.fromUnsafe(() => {
+        val ss = new Settings()
+        val command = new CompilerCommand(scalacOptions, ss)
+        val settings = command.settings
+        settings.YpresentationAnyThread.value = true
+        settings.classpath.value = validatedClasspath.syntax
+        val reporter = new StoreReporter()
```

Comment:
out of curiosity: is this block evaluated lazily on purpose to capture some state that it not available eagerly?

Reply:
So the creation of Global was lazy, I just put the entire block as lazy. 
Usually, if the semanticdb file is present, global is never instantiated. 
I don't think there is any state that is captured later..